ref_builder/plan.py

```diff
@@ -28,34 +42,70 @@ def __str__(self) -> str:
         return f"{self.prefix} {self.key}"
 
 
-class Segment(BaseModel):
-    """The metadata of the segment."""
+class SequencePlan(BaseModel):
+    """Metadata and expected properties for an included sequence."""
 
     id: UUID4
-    """The unique id number of this segment"""
+    """The unique id number of the sequence plan"""
```

Comment:
This makes me think `Segment` and `MonopartitePlan` shouldn't share an ancestor...
```suggestion
    """The unique ID of the segment or monopartite plan."""
```